nginx.conf

```diff
@@ -156,4 +143,41 @@ http {
         }
 
     }
+
+    server {
+        listen 80;
+        charset UTF-8;
+
+        server_name liste.ndla.no
+                    ~^liste.(.+\.)?ndla.no$;
+
+        add_header X-Cache-Status $upstream_cache_status;
+        error_log /var/log/nginx/error.log debug;
+        access_log /var/log/nginx/access.log;
+
+        include '/listing-frontend-hostname.conf';
```

Comment:
Har du glømt denne?

Reply:
Nei, den includen må være der for det er sånn vi setter `$frontend` variablen som brukes i `proxy_pass` på linje 179.

Reply:
Hva som ligger i filen settes i `start-nginx.sh` og er forskjellig basert på om vi er i k8s miljø eller ei.

Reply:
aha. berre eg som oversåg den linja i start-nginx.sh